.prettierrc.json

```diff
@@ -0,0 +1,7 @@
+{
+    "singleQuote": true,
+    "trailingComma": "es5",
+    "bracketSpacing": true,
+    "tabWidth": 4,
```

Comment:
`tabWidth` should be 2, now there are a lot of changes in this pull request because the formatting settings changed. Please change `tabWidth` to 2 and format all the files you changed agian.